code/modules/admin/verbs/striketeam.dm

```diff
@@ -95,7 +107,7 @@ GLOBAL_VAR_INIT(sent_strike_team, 0)
 			var/mob/living/carbon/human/new_commando = create_death_commando(L, is_leader)
 			new_commando.mind.key = ghost_mob.key
 			new_commando.key = ghost_mob.key
-			new_commando.internal = new_commando.s_store
+			new_commando.internal = new_commando.l_store
```

Comment:
Same as above 
```suggestion
			new_commando.internal = locate(/obj/item/tank/internals in new_commando)
```
but please do check that it actually works...

Reply:
This doesn't work but I don't think `new_commando.internal = new_commando.l_store` ever did either, gonna remove both of those since deathsquad never starts in space and they can enable their internals themself